backend/src/main/scala/cromwell/backend/package.scala

```diff
@@ -35,6 +35,9 @@ package object backend {
                                   key: BackendJobDescriptorKey,
                                   inputs: Map[LocallyQualifiedName, WdlValue]) {
     val call = key.call
+
+    /** The abort functionality potentially becomes available after the job descriptor is created */
+    var abortFunction: Option[() => Unit] = None
```

Comment:
I'll see what I can do...  this particular bit of code was written toward the beginning of this endeavor and has some roots in the way we used to do it in the olde world, so perhaps there's a better way